assets/src/dashboard/components/pill/index.js

```diff
@@ -43,46 +50,61 @@ Source: https://polished.js.org/docs/#hidevisually
 `;
 
 const PillContainer = styled.label`
-  display: inline-flex;
-  justify-content: center;
-  font-family: ${({ theme }) => theme.fonts.pill.family};
-  font-weight: ${({ theme }) => theme.fonts.pill.weight};
-  font-size: ${({ theme }) => `${theme.fonts.pill.size}px`};
-  line-height: ${({ theme }) => theme.fonts.pill.lineHeight}px;
-  letter-spacing: ${({ theme }) => theme.fonts.pill.letterSpacing}em;
+  ${({ theme }) => `
+    display: inline-flex;
+    justify-content: center;
+    font-family: ${theme.fonts.pill.family};
+    font-weight: ${theme.fonts.pill.weight};
+    font-size: ${theme.fonts.pill.size}px;
+    line-height: ${theme.fonts.pill.lineHeight}px;
+    letter-spacing: ${theme.fonts.pill.letterSpacing}em;
+  `}
 `;
+
 const PillLabel = styled.span`
```

Comment:
Since this is a span we will need to add a focus style to it as well.

Reply:
great call out here! So, the span here is completely decorative. The actual element doing the update is the input - which gives us keyboard and voice over help. I'm just hiding the input and using a span inside it to style since it's visually kind of complex. 

I had removed the `:focus` outline previously thinking it was unnecessary for the time being, but this review makes me aware that it's hard to tell what's going on in this component without it. 

SO, I fixed the outline so you can see what you're tabbing through and I added a 'close' button that shows when you tab through at the start so you don't have to get trapped tabbing alllll the way through however many inputs there are in the group before being allowed to exit. 

I hope this helps. 
![keyboard category selection demo](https://user-images.githubusercontent.com/10720454/81316895-05097000-9041-11ea-8da7-1c888c36a16a.gif)